.github/workflows/pylint.yml

```diff
@@ -1,27 +1,27 @@
-#name: Pylint
-#
-#on: [ push, pull_request ]
-#
-#jobs:
-#  build:
-#    runs-on: ubuntu-latest
-#    strategy:
-#      matrix:
-#        python-version: [ "3.8" ]
-#    steps:
-#      - uses: actions/checkout@v2
-#      - name: Set up Python ${{ matrix.python-version }}
-#        uses: actions/setup-python@v2
-#        with:
-#          python-version: ${{ matrix.python-version }}
-#      - name: Install dependencies
-#        run: |
-#          python -m pip install --upgrade pip
-#          pip install -r requirements-dev.txt
-#      - name: Check if PR
-#        if: github.event.pull_request.number
-#        run: |
-#          echo "enable=fixme" >> .pylintrc
-#      - name: Analysing the code with pylint
-#        run: |
-#          pylint $(git ls-files '*.py')
+name: Pylint
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
```

Comment:
Why is a strategy matrix used here? This job is only being ran on one Python version